app/Services/AuthenticationService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
+
+use Brewmap\Eloquent\User;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Laravel\Socialite\Contracts\User as FacebookUser;
+
+class AuthenticationService
+{
+    /**
+     * @throws HttpResponseException
+     */
+    public function login(Request $request): string
+    {
+        $user = User::where("email", $request->email)->first();
```

Comment:
Please use `::query()` builder on model queries:
```
User::query()->where()->...
```
It's helpful when you're using IDE method completions.